src/main/java/mate/academy/exception/AuthenticationException.java

```diff
@@ -0,0 +1,6 @@
+package mate.academy.exception;
+
+public class AuthenticationException extends RuntimeException {
+    public AuthenticationException(String s) {
```

Comment:
The constructor should pass the message `s` to the superclass constructor by calling `super(s);`. This ensures that the exception message is properly handled and accessible.